src/graph.h

```diff
@@ -217,6 +217,18 @@ struct Edge {
   bool is_phony() const;
   bool use_console() const;
   bool maybe_phonycycle_diagnostic() const;
+
+  // May be actual mtime as well as just 0
```

Comment:
Please use `///` here. IDEs will use the comment as a documentation for the variable then.

Reply:
Thanks, done. I also added another comment, but with double `//` slashes because it documents why the `EdgeComparator` sorts like this. See if you want me to change it to `///`, I've used the `//` because I see such kind of comment in other places uses the double slashes.